app/pods/agenda/agendaitems/agendaitem/documents/template.hbs

```diff
@@ -1,56 +1,67 @@
-<div class="auk-u-mb-10">
-  <div class="auk-u-mb-4">
-    <div class="auk-u-flex auk-u-flex--vertical-center">
-      <h4 class="auk-h3 auk-u-m-0">
+<WebComponents::AuToolbar class="auk-u-mb-4">
+  <WebComponents::AuToolbar::Group @position="left">
+    <WebComponents::AuToolbar::Item>
+      <h4 class="auk-toolbar__title">
         {{t "documents"}}
       </h4>
-      {{#if (and this.currentSession.isEditor this.model.pieces.length)}}
-        <div class="auk-u-ml-2">
-          <button data-test-subcase-documents-edit class="auk-button-link" type="button"
-            {{on "click" this.enablePieceEdit}}>
-            {{t "edit"}}
-          </button>
-        </div>
-      {{/if}}
-    </div>
+    </WebComponents::AuToolbar::Item>
+    {{#if (and this.currentSession.isEditor this.model.pieces.length)}}
+      <WebComponents::AuToolbar::Item>
+        <button data-test-subcase-documents-edit class="auk-button-link" type="button"
+          {{on "click" this.enablePieceEdit}}>
+          {{t "edit"}}
+        </button>
+      </WebComponents::AuToolbar::Item>
+    {{/if}}
+  </WebComponents::AuToolbar::Group>
+  <WebComponents::AuToolbar::Group @position="right">
+    {{#if (and this.currentSession.isOvrb this.model.pieces)}}
+      <WebComponents::AuToolbar::Item>
+        <WebComponents::AuButton data-test-subcase-documents-publish type="button"
```

Comment:
Deze button, waarom heeft die `type=button` ?
Zit al standaard in de au-button template